packages/tabs-react/src/useTabNavigation.ts

```diff
@@ -0,0 +1,106 @@
+import { useCallback, useEffect, useState } from "react";
+
+const slugify = (input: string) =>
```

Comment:
@wkillerud jeg ser du har dratt inn `@sindresorhus/slugify` i portaleksemplene, men nøler litt med å legge til eksterne avhengigheter i pakkene. Dette er muligens bra nok for de fleste av våre tilfeller?

Reply:
Det burde dekke det ja :)